Packs/DeveloperTools/ReleaseNotes/1_3_0.md

```diff
@@ -0,0 +1,3 @@
+
+### Developer Tools
```

Comment:
I think it is two sorry
```suggestion
## Developer Tools
```

Reply:
See this https://jira-dc.paloaltonetworks.com/browse/CIAC-8262

Reply:
thanks!